app.py

```diff
@@ -21,41 +22,53 @@
     authority=app.config.get("AUTHORITY"),
     client_id=app.config["CLIENT_ID"],
     client_credential=app.config["CLIENT_SECRET"],
-    )
+)
+
 
 @app.route("/login")
 def login():
-    return render_template("login.html", version=identity.__version__, **auth.log_in(
-        scopes=app_config.SCOPE,  # Have user consent scopes during log-in
-        redirect_uri=url_for("auth_response", _external=True),  # Optional. If present, this absolute URL must match your app's redirect_uri registered in Azure Portal
-        ))
+    result = auth.log_in(
```

Comment:
Would you consider NOT introducing a helper variable `result`? Reason being, your change `result = auth.log_in(...)` reads like the function yields an auth result. But it does not. It just returns some "things" to be rendered to end user. The real auth result will only be available in next function's `result = auth.complete_log_in(...)`.

Alternatively, you could choose a different name other than "result". But, naming is hard in computer science, that is why the preexisting implementation uses `return render_template("login.html", version=identity.__version__, **auth.log_in(` (which is still reasonably short, by the way).